### DISCUSSION TITLE
nzyme-tap.conf: remove unused WiFi interfaces recommended?

### DISCUSSION THREAD

**Discussion by sudo-joe** [nzymedefense/nzyme#1149] [Questions]
Hi

"wifi_interfaces.wlx00c0cab043da" is my Alpha AWUS036ACM card. Should the second (non existent) wifi-interface  
wlx00c0ca000001 (with all its 6 lines) be removed from nzyme-tap.conf:?

Thank you!

[wifi_interfaces.wlx00c0cab043da]
active = true
channel_width_hopping_mode = "full" # Learn more: https://go.nzyme.org/wifi-hopping-channel-width-mode
channels_2g = [1, 6, 11]
channels_5g = [100, 102, 104, 106, 108, 110, 112, 136, 138, 140, 149, 151, 153, 155, 157, 159, 161, 165]
channels_6g = []

[wifi_interfaces.wlx00c0ca000001]
active = true
channel_width_hopping_mode = "full" # Learn more: https://go.nzyme.org/wifi-hopping-channel-width-mode
channels_2g = [2, 3, 4, 5, 7, 8, 9, 10, 12, 13]
channels_5g = [36, 40, 44, 48, 52, 56, 60, 64, 173]
channels_6g = [5, 9, 13, 17, 21, 25, 29, 109, 113, 153, 157, 161, 165, 169, 173, 177, 181, 185, 189, 229, 233]

**Comment by lennartkoopmann** [comment]
Yes, you should not configure WiFi interfaces that you don't have.

**Comment by sudo-joe** [comment]
Thank's Lennart for the help and sorry that I have to ask such  basic questions:

1.
In order to "activate" an interface, should only the Brackets be removed? Like that?
wifi_interfaces.wlx00c0cab043da

2. 
And if the interface is not used, should all the 6 text lines be removed?

Thank's !!

**Reply by lennartkoopmann** [reply]
I'd recommend to look at the example configuration file and study the syntax.
